src/Components/Nav.js

```diff
@@ -1,7 +1,108 @@
 import React, { Component } from 'react';
+import styled from 'styled-components';
 
-export default class Nav extends Component {
-  render() {
-    return <div></div>;
+const Nav = () => {
+  return (
+    <NavContainer>
+      <Nav1stLine>
+        <LogoAndSearchBar>
+          <LogoImg alt="watta_logo" src="/images/wa-ttaLogo_0.5.png" />
+          <LogoText inputColor="#495056">Wa:tta Taxi</LogoText>
+          <SearchBar
+            type="text"
+            placeholder="가고 싶은 역을 검색해보세요"
+          ></SearchBar>
```

Comment:
```suggestion
          <SearchBar
            type="text"
            placeholder="가고 싶은 역을 검색해보세요"
          />
```
셀프 클로징 태그 사용할 수 있습니다